app/views/data_centers/_data_center_list.html.erb

```diff
@@ -0,0 +1,11 @@
+<% @results[:results].each do |dc_key, dc_data| %>
+  <tr>
+    <td><%= link_to dc_key, data_center_url(network: params[:network], key: dc_key.gsub('/', '-slash-')) %></td>
+    <td><%= dc_data[:aso] %></td>
+    <td><%= dc_data[:count] %></td>
+    <td>
+      <%= number_with_delimiter lamports_to_sol(dc_data[:active_stake]).round(0) %>
+      (<%= number_to_percentage((dc_data[:active_stake] / @results[:total_stake].to_f) * 100.0, precision: 2) %>)
```

Comment:
`(dc_data[:active_stake] / @results[:total_stake].to_f) * 100.0` could be a helper method